test/syn_registry_SUITE.erl

```diff
@@ -104,6 +104,8 @@ groups() ->
         ]},
         {four_nodes_registry, [shuffle], [
             four_nodes_concurrency
+            one_node_strict_mode,
+            one_node_repeated_restart
```

Comment:
if inside of this group, we have a four nodes setup, can you please rename this with the convention of {number_of_nodes}_{test_name}?

Reply:
My apologies, a mistake in partial staging. 
Fixed https://github.com/ostinelli/syn/compare/58cb9ea1b9ab7db52d73c680c4c16ed3702c55e5..872f137c5da0fc4978e1e6fe545b33c058ca09f6